### DISCUSSION TITLE
Any site describing Digital

### DISCUSSION THREAD

**Discussion by Lisiczny** [hneemann/Digital#917] [Q&A]
Do you know any sites, which describes this program in more educational way? I'm on my way to finish mine Engineering work about design of digital circuits, specially in Digital program. I will appreciate any help :)

**Comment by hneemann** [comment]
There is not much to say about the simulator. 
The operation is (almost) self-explanatory, as there are tooltips almost everywhere. The individual components are simulated as you would expect, with a few special cases that are not really of interest to the beginner. When the technical details of the simulation are required there is nothing but the source code and this forum. 
When it comes to the circuits themselves, any undergraduate textbook will do.

If there are any questions, they can always be asked here. I'm glad to help.
